### PR TITLE
修复 paddle.nn.Conv3DTranspose 等 API 的文档

### PR DESCRIPTION
**Pull request by LearningPawn**
按 [**在线文档**](https://shimo.im/sheets/e1Az48XnO4t6g7qW/dfHaD) 的 336-345 行，编号为 304-313 的要求进行了如下修改：

1. 修改 docs/api/paddle/nn/Conv3DTranspose_cn.rst、docs/api/paddle/nn/functional/conv3d_transpose_cn.rst **注意**的格式，中英文标点修正，optional -> 可选；
2. 修改 docs/api/paddle/nn/CTCLoss_cn.rst、docs/api/paddle/nn/functional/ctc_loss_cn.rst 中的格式，错误的参数名 `logits`，补充缺少的参数 `norm_by_times`；
3. 修改其余文件的格式、该 OP、中英文间空格等问题。

_关于 docs/api/paddle/nn/Conv3DTranspose_cn.rst 中 output_size 的说明：_
- 通过查看 [Conv3DTranspose 的原代码](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/nn/layer/conv.py#L1215) ，会发现 note 中提到的 output_size 事实上是在该类的 forward 函数中指定的，但中文文档中未提及该函数，直接对 output_size 进行说明可能会让读者感觉突兀，故建议添加相关函数的简单说明。

涉及到的 API 有

- paddle.nn.Conv3DTranspose
- paddle.nn.functional.conv3d_transpose
- paddle.nn.CTCLoss
- paddle.nn.functional.ctc_loss
- paddle.nn.Dropout
- paddle.nn.functional.dropout
- paddle.nn.Dropout2D
- paddle.nn.functional.dropout2d
- paddle.nn.Dropout3D
- paddle.nn.functional.dropout3d

对应的英文文档修复为：https://github.com/PaddlePaddle/Paddle/pull/49149

### CODE REVIEW

File: docs/api/paddle/nn/Conv3DTranspose_cn.rst [hunk 1]
```diff
@@ -6,11 +6,10 @@ Conv3DTranspose
 .. py:class:: paddle.nn.Conv3DTranspose(in_channels, out_channels, kernel_size, stride=1, padding=0, output_padding=0, groups=1, dilation=1, weight_attr=None, bias_attr=None, data_format="NCDHW")
 
 
-三维转置卷积层（Convlution3d transpose layer)
+三维转置卷积层（Convlution3d transpose layer）
 
-该层根据输入（input）、卷积核（kernel）和卷积核空洞大小（dilations）、步长（stride）、填充（padding）来计算输出特征层大小或者通过 output_size 指定输出特征层大小。输入(Input)和输出(Output)为 NCDHW 或者 NDHWC 格式。其中 N 为批尺寸，C 为通道数（channel），D 为特征深度，H 为特征层高度，W 为特征层宽度。转置卷积的计算过程相当于卷积的反向计算。转置卷积又被称为反卷积（但其实并不是真正的反卷积）。欲了解卷积转置层细节，请参考下面的说明和 参考文献_。如果参数 bias_attr 不为 False，转置卷积计算会添加偏置项。
+该层根据输入（input）、卷积核（kernel）和卷积核空洞大小（dilations）、步长（stride）、填充（padding）来计算输出特征层大小或者通过 output_size 指定输出特征层大小。输入（Input）和输出（Output）为 NCDHW 或者 NDHWC 格式。其中 N 为批尺寸，C 为通道数（channel），D 为特征深度，H 为特征层高度，W 为特征层宽度。转置卷积的计算过程相当于卷积的反向计算。转置卷积又被称为反卷积（但其实并不是真正的反卷积）。欲了解卷积转置层细节，请参考下面的说明和 `参考文献 Deconvolutional Networks <https://www.matthewzeiler.com/mattzeiler/deconvolutionalnetworks.pdf>`_。如果参数 bias_attr 不为 False，转置卷积计算会添加偏置项。
 
-.. _参考文献：https://www.matthewzeiler.com/mattzeiler/deconvolutionalnetworks.pdf
```
Comment (Ligoml) [inline docs/api/paddle/nn/Conv3DTranspose_cn.rst:13]: 其实这两种索引方式都可以，只是上面的没用对，具体用法可以参考：[飞桨文档相互引用](https://github.com/PaddlePaddle/docs/wiki/%E9%A3%9E%E6%A1%A8%E6%96%87%E6%A1%A3%E7%9B%B8%E4%BA%92%E5%BC%95%E7%94%A8)

File: docs/api/paddle/nn/Conv3DTranspose_cn.rst [hunk 2]
```diff
@@ -21,45 +20,44 @@ Conv3DTranspose
...
+    - 输入：:math:`(N, C_{in},  H_{in}, W_{in})`
 
     - 卷积核：:math:`(C_{in}, C_{out}, K_{d}, K_{h}, K_{w})`
 
     - 偏置：:math:`(C_{out})`
 
-    - 输出：:math:`（N，C_{out}, H_{out}, W_{out}）`
+    - 输出：:math:`(N, C_{out}, H_{out}, W_{out})`
 
     其中
```
Comment (Ligoml) [inline docs/api/paddle/nn/Conv3DTranspose_cn.rst:64]: 这里的公式部分中英文文档是不一致的，@jerrywgz 看下哪边是对的？

File: docs/api/paddle/nn/functional/dropout2d_cn.rst [hunk 3]
```diff
@@ -5,16 +5,17 @@ dropout2d
...
 .. note::
-   该 op 基于 ``paddle.nn.functional.dropout`` 实现，如您想了解更多，请参见 :ref:`cn_api_nn_functional_dropout` 。
+   基于 ``paddle.nn.functional.dropout`` 实现，如您想了解更多，请参见 :ref:`cn_api_nn_functional_dropout` 。
 
 参数
 :::::::::
- - **x** (Tensor)：形状为[N, C, H, W]或[N, H, W, C]的 4D `Tensor`，数据类型为 float32 或 float64。
- - **p** (float)：将输入通道置 0 的概率，即丢弃概率。默认：0.5。
- - **training** (bool)：标记是否为训练阶段。默认：True。
+ - **x** (Tensor)：形状为 [N, C, H, W] 或 [N, H, W, C] 的 4D `Tensor`。数据类型为 float32 或 float64。
```
Comment (Ligoml) [inline docs/api/paddle/nn/functional/dropout2d_cn.rst]:
```suggestion
 - **x** (Tensor) - 形状为 [N, C, H, W] 或 [N, H, W, C] 的 4D `Tensor`。数据类型为 float32 或 float64。
```

File: docs/api/paddle/nn/functional/dropout2d_cn.rst [hunk 4]
```diff
@@ -5,16 +5,17 @@ dropout2d
 
 .. py:function:: paddle.nn.functional.dropout2d(x, p=0.5, training=True, name=None)
 
-根据丢弃概率 `p`，在训练过程中随机将某些通道特征图置 0(对一个形状为 `NCHW` 的 4 维 Tensor，通道特征图指的是其中的形状为 `HW` 的 2 维特征图)。
+根据丢弃概率 `p`，在训练过程中随机将某些通道特征图置 0 (对一个形状为 `NCHW` 的 4 维 Tensor，通道特征图指的是其中的形状为 `HW` 的 2 维特征图)。
 
 .. note::
-   该 op 基于 ``paddle.nn.functional.dropout`` 实现，如您想了解更多，请参见 :ref:`cn_api_nn_functional_dropout` 。
+   基于 ``paddle.nn.functional.dropout`` 实现，如您想了解更多，请参见 :ref:`cn_api_nn_functional_dropout` 。
```
Comment (Ligoml) [inline docs/api/paddle/nn/functional/dropout2d_cn.rst]: 这个note似乎不太有必要？要不参考英文文档直接改成文本吧

Comment (Ligoml) [reply docs/api/paddle/nn/functional/dropout2d_cn.rst]: 另外英文文档那边需要给 paddle.nn.functional.dropout 加上超链接

Comment (Ligoml) [reply docs/api/paddle/nn/functional/dropout2d_cn.rst]: paddle.nn.Dropout3D 和 paddle.nn.functional.dropout3d 同

**Comment by LearningPawn** [comment]
@Ligoml 感谢您的 Review，现已根据您的意见进行了修改：
1. [Conv3DTranspose](http://preview-pr-5526.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/Conv3DTranspose_cn.html) 的参考文献格式已纠错；公式参考[前文中的表述](https://github.com/PaddlePaddle/docs/pull/5526/files#diff-2013304695a79d0a1f843e5e923aa4b8338484dff0a8ee5bc52947e0c852df58L22)如下，应该是英文版是正确的，现已修改；
> 其中：
> &nbsp; X: 输入，具有 NCDHW 或 NDHWC 格式的 5-D Tensor
2. [dropout2d](http://preview-pr-5526.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/functional/dropout2d_cn.html) 参数格式已更改；note 已参考英文文档改为文本。
3. [dropout2d_en](http://preview-paddle-pr-49149.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/functional/dropout2d_en.html)、[dropout3d_en](http://preview-paddle-pr-49149.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/functional/dropout3d_en.html)、[Dropout3d_en](http://preview-paddle-pr-49149.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/Dropout3D_en.html) 等英文文档已添加 api 参考链接，等待 [docs_preview](https://github.com/PaddlePaddle/Paddle/pull/49149/commits/b67260415a3a49a002f90da283d7c9be5ac0887e) 结束，应该能看到效果。

**Review by Ligoml — APPROVED** [review]
good job！LGTM for docs
检查预览后可以合入
